### ISSUE TITLE
Anchor Solid Infill by X mm from SuperSlicer

### ISSUE THREAD

**Issue by djngnear** [bambulab/BambuStudio#708]
**Is your feature request related to a problem? Please describe.**
I love the option to fill in small areas with solid infill on top surfaces like under text or smaller objects.  I go to Superslicer for this very reason. By default it's set to 150% but I like to bump it up to 300 or 400%. That way the monotonic infill just looks better and it doesn't need to navigate around holes.
**Describe the solution you'd like**
Implement the option to change the solid infill percentage like in SuperSlicer.
**Describe alternatives you've considered**

**Additional context**
![SS Advanced Infill Settings](https://user-images.githubusercontent.com/87901234/203667040-0c5560ef-37c4-447b-a341-fd7fa0e36a2b.JPG)

**Comment by user897943** [comment]
Can you post two photos, showing without this option and with your option at your preferred setting.

**Comment by djngnear** [comment]
Yeah so here is a sign with the default setting of 150% anchor solid infill (1st picture). 
And what it looks like at the top surface layer (2nd picture). 
By changing that setting to 300%, there's no more holes in the top surface to navigate around (3rd & 4th picture).
I think this setting is super useful for an extra nice top layer under lettering and smaller geometry.

![SS Anchor Infill 1](https://user-images.githubusercontent.com/87901234/204912781-a71b1443-e125-4f5a-8855-5e14cc191f00.JPG)
![SS Anchor Infill 2](https://user-images.githubusercontent.com/87901234/204912770-deaaec0f-843d-4da0-ba14-899b0863ef11.JPG)
![SS Anchor Infill 3](https://user-images.githubusercontent.com/87901234/204912778-79f21cc1-4c1e-49df-aace-417bb7f409bc.JPG)
![SS Anchor Infill 4](https://user-images.githubusercontent.com/87901234/204912779-d01c8e90-2809-4e1f-ab67-db2365d5ad13.JPG)

**Comment by user897943** [comment]
Thanks for this, it should definitely be in Bambu Studio. It would add strength in some situations I can think of. But not sure if they should name it like that... Just from reading I can't really make out what "anchor solid infill by" means, your example is great. What would you call the feature if you had to give it another name that's easier to understand?

**Comment by AbeFM** [comment]
Outer (top) surface small hole solid infill ?

Yeah, I like what he's asking for!

**Comment by dom2587** [comment]
im also looking for this setting to be enabled, i'm trying to print the head(amame) headset and there cones require the ability to set this to 0 for it to print correctly.

**Comment by djngnear** [comment]
> im also looking for this setting to be enabled, i'm trying to print the head(amame) headset and there cones require the ability to set this to 0 for it to print correctly. 

Could you take a screenshot of what you're referring to? I'm interested.

**Comment by dom2587** [comment]
> 

sure thing https://imgur.com/a/nZ7JCm8

if you look at the infill in bambu and compare it to the prusa slicer that is the infill anchor setting at play, why the bambu doesnt get the solid walls from the 3mf while prusa does is my next problem to sort lol.

**Comment by djngnear** [comment]
Wow, yeah I see that. Hopefully they'll address it soon.

**Comment by djngnear** [comment]
Please look into this!

**Comment by SaltWei** [comment]
> im also looking for this setting to be enabled, i'm trying to print the head(amame) headset and there cones require the ability to set this to 0 for it to print correctly.

This can be set in latest bambustudio 
![image](https://github.com/bambulab/BambuStudio/assets/110660407/d3ac2b93-3c8a-4465-b2d4-9dfbecae42b6)

**Comment by SaltWei** [comment]
> Implement the option to change the solid infill percentage like in SuperSlicer.

We will evaluate this feature and add it into developer mode. Thanks everyone!

**Comment by djngnear** [comment]
> > im also looking for this setting to be enabled, i'm trying to print the head(amame) headset and there cones require the ability to set this to 0 for it to print correctly.
> 
> 
> 
> This can be set in latest bambustudio 
> 
> ![image](https://github.com/bambulab/BambuStudio/assets/110660407/d3ac2b93-3c8a-4465-b2d4-9dfbecae42b6)
> 
> 
Yeah, that's for the sparse infill
